roles/accprovision/tasks/process.yaml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Create temporary directory for acc-provision run
+  file:
+    path: "{{ acc_provision_dir_path }}"
+    state: directory
+    mode: '0777'
```

Comment:
Do we need all access - Might be 644?

Reply:
We need to create and read files in that directory

Reply:
777 is for whole access. like running a script. For directory I believe 755 also works

Reply:
Changed it, works!